var/spack/repos/builtin/packages/mirdeep/package.py

```diff
@@ -31,9 +31,9 @@ class Mirdeep(Package):
        by analyzing sequenced RNAs."""
 
     homepage = "https://www.mdc-berlin.de/8551903/en/"
-    url      = "https://www.mdc-berlin.de/45995549/en/research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/mirdeep2_0_0_8.zip"
+    url      = "https://www.mdc-berlin.de/media/16193"
```

Comment:
If you fetch from this URL, it redirects to:
https://www.mdc-berlin.de/system/files/migrated_files/fiona/mirdeep2_0_0_8.zip

I think we should use this URL instead. It makes it easy for Spack to download multiple versions from the same URL because there isn't some random int in there.